src/main/java/at/ac/tuwien/kr/alpha/grounder/transformation/impl/ChoiceHeadToNormal.java

```diff
@@ -25,34 +25,42 @@
  * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package at.ac.tuwien.kr.alpha.grounder.transformation;
+package at.ac.tuwien.kr.alpha.grounder.transformation.impl;
```

Comment:
Just for completeness: the copyright line above `Copyright (c) 2017-2018, the Alpha Team.` should be to `2019`.

Reply:
done